compiler/AST/build.cpp

```diff
@@ -1360,7 +1360,7 @@ buildStandaloneForallLoopStmt(Expr* indices,
  *   }
  */
 BlockStmt*
-buildForallLoopStmt(Expr*      indices,
+buildForallLoopStmtOld(Expr*   indices,
```

Comment:
I think you said this is still used for forall expressions? Could you add a comment indicating what causes it to be called and what related code would need to change in order to remove it? Could you add those topics as future work in the PR message?